addons/events/fnc_playerEvent.sqf

```diff
@@ -44,7 +44,7 @@ private _state = [
     _unit, group _unit, leader _unit,
     currentWeapon _unit, currentMuzzle _unit, currentWeaponMode _unit,
     getUnitLoadout _unit, _vehicle, _turret, _vehicle currentWeaponTurret _turret,
-    currentVisionMode _controlledEntity, cameraView
+    currentVisionMode _controlledEntity, cameraView, getTurretOpticsMode _unit
```

Comment:
I wonder if this should be `_unit` or `focusOn` for UAVs
will test

Reply:
doesn't seem to matter
`getTurretOpticsMode focuson` = `getTurretOpticsMode player`